### PR TITLE
Add `useAt` option to autofix `.lastObject` to `.at(-1)` in `no-get` rule

### PR DESCRIPTION
**Pull request by ArtixZ**
`.lastObject` autofix rule can be simplified as `.lastObject` ==> `.at(-1)`. 
We can lift the limit of handling only one `.lastObject` in path as well.

### CODE REVIEW

**Comment by bmish** [comment]
This is a nice improvement. However, I don't think we can make this the default behavior until Node 16 becomes end-of-life in Sept, 2023. We could put this behind an option `useAt` (similar to `useOptionalChaining`) until then.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at
https://github.com/nodejs/Release

**Comment by ArtixZ** [comment]
@bmish Thanks for the advice. I'd assume the default value of `useAt` should be `true`, is it?

**Comment by bmish** [comment]
We need to keep this feature turned off by default until we drop the corresponding Node version support. So `useAt` option default to `false` for now.

File: lib/rules/no-get.js [hunk 1]
```diff
@@ -195,6 +204,7 @@ module.exports = {
     const useOptionalChaining = context.options[0] && context.options[0].useOptionalChaining;
     const catchSafeObjects = context.options[0] ? context.options[0].catchSafeObjects : true;
     const catchUnsafeObjects = context.options[0] && context.options[0].catchUnsafeObjects;
+    const useAt = context.options[0] && context.options[0].useAt;
```
Comment (bmish) [inline lib/rules/no-get.js:207]: Let's keep this list of variables alphabetical.

File: docs/rules/no-get.md [hunk 2]
```diff
@@ -98,6 +98,7 @@ This rule takes an optional object containing:
 - `boolean` -- `useOptionalChaining` -- whether the rule should use the [optional chaining operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) `?.` to autofix nested paths such as `this.get('some.nested.property')` to `this.some?.nested?.property` (when this option is off, these nested paths won't be autofixed at all) (default `true`)
 - `boolean` -- `catchSafeObjects` -- whether the rule should catch non-`this` imported usages like `get(foo, 'bar')` (default `true`)
 - `boolean` -- `catchUnsafeObjects` -- whether the rule should catch non-`this` usages like `foo.get('bar')` even though we don't know for sure if `foo` is an Ember object (default `false`)
+- `boolean` -- `useAt` -- whether the rule should use `at(-1)` to replace `lastObject` (default `false`)
```
Comment (bmish) [inline docs/rules/no-get.md:101]: Let's keep this list of options alphabetical.

Comment (bmish) [inline docs/rules/no-get.md:101]: Two tweaks:
* Link to the doc about this
* Add TODO reminder to enable it

```suggestion
- `boolean` -- `useAt` -- whether the rule should use `at(-1)` [Array.prototype.at()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at) to replace `lastObject` (default `false`) (TODO: enable by default once we require Node 16.6)
```

**Comment by ArtixZ** [comment]
Sorry I was sidetracked. Will have an update today.

**Review by bmish — COMMENTED** [review]
Thanks!

**Review by bmish — APPROVED** [review]

**Pull request merged**